app/src/main/java/com/example/campusguide/DirectionsActivity.kt

```diff
@@ -237,9 +273,9 @@ class DirectionsActivity : AppCompatActivity(), AdapterView.OnItemClickListener
     }
 
     private fun onTravelModeClicked(travelMode: String, paths: Map<String, PathPolyline>) {
-        showMap()
         removePreviousPath()
         currentPath = paths.getValue(travelMode)
         setPathOnMapAsync(currentPath)
+        showMap()
```

Comment:
Is there any reason the call to showMap was put at the end of the function?

Reply:
It is a personal choice. It feels more fluid when the map get loaded with the path already on the map.